src/averageAges.js

```diff
@@ -15,12 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+  const men = people.filter(person => person.sex === 'm');
+  let menFromCentury;
+
+  century !== undefined
```

Comment:
You don't need to explicitly check if it's undefined
```suggestion
  century
```